k6/models/payments.js

```diff
@@ -4,7 +4,22 @@ import http from 'k6/http';
 import config from './config.js';
 const { baseUrl } = config;
 
-export default class paymentsModel {
+export default class PaymentsModel {
+  verifyPaymentDryRun(programId, amount) {
```

Comment:
I am not sure if I understand it correctly. But should you not also use verifyPaymentDryRunUntilSuccess here?